### ISSUE TITLE
Unable to login to OSIO (prob. after I was added as a collaborator on the openshiftio space)

### ISSUE THREAD

**Issue by tisnik** [openshiftio/openshift.io#3670]
Hi team,

I'm no longer able to login to OSIO. It worked fine, but after I was added as a collaborator on the openshiftio space, the login page displays:

Account already exists
User with email ptisnovs@redhat.com already exists. How do you want to continue?

And neither option works there.

[https://sso.openshift.io/auth/realms/fabric8/login-actions/first-broker-login?client_id=fabric8-online-platform]

Is there any change to get some logs to see what happens? Or to reset the account?

**Comment by alexeykazakov** [comment]
yes. there are two developers.redhat.com (and openshift online) accounts with the same email. It causes problems. I deleted the unused account from OSIO. Try to login now. If not working then need more details. Do you use your username or email when logging?
